LanguageFeatures/Private-named-parameters/super_parameters_A01_t01.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A super parameter generates an implicit argument that forwards to
+/// a superclass constructor. The super constructor's argument name is always
+/// public, even if the corresponding constructor parameter uses this feature
+/// and has a private name. Thus, super parameters continue to always use public
+/// names.
+///
+/// @description Check that private super parameter can be invoked using its
+/// public name.
```

Comment:
I think it would make sense (and cover a slightly different amount of code in the tools) to have a similar test where the superinitializer is written explicitly:

```dart
class B1 extends A1 {
  B1(String p) : super(p: p);
  B1.named(String p) : super(p: p);
}
```